src/plugins/home/public/application/components/_solutions_section.scss

```diff
@@ -1,116 +1,99 @@
-.homSolutionsSection {
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.homSolutions {
   margin-top: -$euiSizeXL;
   min-height: $euiSize * 16;
-  display: flex;
 
-  .homSolutionsSection__panel {
-    display: flex;
-    align-items: stretch;
-
-    .homSolutionsSection__panelHeader {
-      border-radius: $euiBorderRadius 0 0 $euiBorderRadius;
-      color: $euiColorEmptyShade;
-
-      @include euiBreakpoint("xs", "s") {
-        border-radius: $euiBorderRadius $euiBorderRadius 0 0;
-      }
-    }
-
-    .homSolutionsSection__panelHeaderContent {
-      padding: $euiSize;
-      color: $euiColorGhost;
-    }
+  @include euiBreakpoint('xs', 's') {
+    flex-direction: column;
+  }
+}
 
-    .homSolutionsSection__panelIcon {
-      margin: 0 auto $euiSizeS;
-      padding: $euiSizeS;
-      background-color: $euiColorGhost;
-      box-shadow: none;
+.homSolutions__group {
+  max-width: 50%;
 
-      .euiIcon__fillNegative {
-        fill: $euiColorInk;
-      }
-    }
+  @include euiBreakpoint('xs', 's') {
+    max-width: none;
+  }
+}
 
-    .homSolutionsSection__panelSubtitle {
-      margin-top: $euiSizeXS;
-    }
+.homSolutionPanel {
+  align-items: stretch; // Necessary because element is a button
+  display: flex !important;
+  flex-direction: column;
+  overflow: hidden;
+}
 
-    .homSolutionsSection__panelDescriptions {
-      height: 100%;
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      padding: $euiSize;
+.homSolutionPanel__header {
+  color: $euiColorEmptyShade;
+  padding: $euiSize;
+}
 
-      @include euiBreakpoint("xs", "s") {
-        height: auto;
-        text-align: center;
-      }
-    }
-  }
+.homSolutionPanel__icon {
+  background-color: $euiColorEmptyShade !important;
```

Comment:
I'm thinking we should treat the card header designs more as branded elements meaning they don't change between light and dark themes. In the dark theme we do adjust colors for added contrast which then gets applied to buttons, icons, etc. In this case however, I feel this lightening leads to a less desirable visual treatment for the cards.
 
I'd like to hear other opinions, but I am in favor of treating the card headers more like brand elements - keeping the richer colors/not changing between themes/hard coding those values. This would include the background color, text color, and icon colors.

For those following along, this PR results in a dark mode that looks like:

<img width="1277" alt="Screenshot 2020-08-03 09 02 20" src="https://user-images.githubusercontent.com/446285/89194969-b602a680-d56d-11ea-852f-c38f88baa239.png">

Prior to this PR, it looks like:

<img width="1212" alt="Screenshot 2020-07-30 12 14 09" src="https://user-images.githubusercontent.com/446285/89195034-ca46a380-d56d-11ea-90aa-097e471317b1.png">

Reply:
While I like the concept of considering these card "branded" to some degree, I do personally prefer the high contrast colored header areas that is provided by using the EUI color variables. I like the attention it demands and the resulting look is one that doesn't offend me. Perhaps the branding concept and changing the colors between light and dark themes need not be mutually exclusive?

Reply:
It's subjective, I know, but on first sight I did not prefer the light shades of green and pink.

We're following the color variables religiously throughout Kibana - and for good reason - but this situation strikes me as a possible exception.